benchmarks/stemming_sample.txt

```diff
@@ -10781,9 +10781,9 @@
 ΜΟΥΣΑΜΑΔΕΣ,ΜΟΥΣΑΜ
 ΜΟΥΣΑΜΑΣ,ΜΟΥΣΑΜ
 ΜΟΥΣΕΙ,ΜΟΥΣ
-ΜΟΥΣΕΙΑ,ΜΟΥΣΕΙ
-ΜΟΥΣΕΙΟ,ΜΟΥΣΕΙ
-ΜΟΥΣΕΙΟΥ,ΜΟΥΣΕΙ
+ΜΟΥΣΕΙΑ,ΜΟΥΣ
+ΜΟΥΣΕΙΟ,ΜΟΥΣ
+ΜΟΥΣΕΙΟΥ,ΜΟΥΣ
```

Comment:
same root όμως, «μουσείο» is [«ιερό των Μουσών, χώρος ποίησης και μουσικής»](http://www.greek-language.gr/greekLang/modern_greek/tools/lexica/triantafyllides/search.html?lq=%CE%BC%CE%BF%CF%85%CF%83%CE%B5%CE%AF%CE%BF&dq=)